<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1302</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>КАЛЬКУЛЯТОР СИСТЕМАТИЧЕСКИХ ОШИБОК ПРИ ЮСТИРОВКЕ</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1201</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>511</width>
       <height>131</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>481</width>
        <height>91</height>
       </rect>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="rowCount">
       <number>2</number>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>70</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>10</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
      <row>
       <property name="text">
        <string>Истина</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Оценка</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>X</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Y</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Z</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Bet, рад</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Eps, рад</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Alf, рад</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>341</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ПАРАМЕТРЫ ПОЗИЦИОН РЛК В ПСК, м, рад</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>40</y>
       <width>531</width>
       <height>131</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>481</width>
        <height>91</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="rowCount">
       <number>2</number>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>70</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>30</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
      <row>
       <property name="text">
        <string>Истина</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Оценка</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>X</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Y</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Z</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Bet, рад</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Eps, рад</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Alf, рад</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>331</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ПАРАМЕТРЫ ПОЗИЦИОН ИУ В ПСК, м, рад</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ИСХОДНЫЕ ДАННЫЕ</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>221</width>
       <height>111</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ПАЛУБНЫЕ УГЛЫ КОРАБЛЯ, рад</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>66</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row/>
      <column>
       <property name="text">
        <string>Qk, град</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Psi, град</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tet, град</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_9">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>180</y>
       <width>221</width>
       <height>111</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>ПОЛОЖЕНИЕ ЦЕЛИ КГСК, м, рад</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>66</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row/>
      <column>
       <property name="text">
        <string>Qц, рад</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Dц, м</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Hц, м</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>190</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>ВЫЧИСЛИТЬ </string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_7">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>1201</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>10</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>КАЖУЩАЯСЯ СК ИУ</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>161</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>УГЛЫ НАВЕДЕНИЯ ИУ, мрад</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>241</width>
        <height>121</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="rowCount">
       <number>3</number>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>75</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>35</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
      <row>
       <property name="text">
        <string>ИСТИНА</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>ОЦЕНКА</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>ОШИБКА</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>ПУГН</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ПУВН</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_11">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>911</width>
       <height>131</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>0</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ДИФФЕРЕНЦИАЛЫ</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>891</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="rowCount">
       <number>2</number>
      </property>
      <property name="columnCount">
       <number>9</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>90</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
      <row>
       <property name="text">
        <string>dBet, мрад</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>dEps,мрад</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>X,м</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Y м</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Z м</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Bet РЛК</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Bet ИУ</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Eps РЛК</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Eps ИУ</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Alf РЛК</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Alf ИУ</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_12">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>1201</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>АБСОЛЮТНАЯ СК ИУ</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>321</width>
       <height>171</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>УГЛЫ НАВЕДЕНИЯ ИУ, мрад</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>281</width>
        <height>131</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="rowCount">
       <number>3</number>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>90</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>35</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
      <row>
       <property name="text">
        <string>ИСТИНА</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>ОЦЕНКА</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>ОШИБКА</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>ПУГН</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ПУВН</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_14">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>40</y>
       <width>761</width>
       <height>131</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>0</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ДИФФЕРЕНЦИАЛЫ</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>731</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="rowCount">
       <number>2</number>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>95</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
      <row>
       <property name="text">
        <string>dBet, мрад</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>dEps,мрад</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>X ИУ,м</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Y ИУ м</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Z ИУ м</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Bet ИУ</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Eps ИУ</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Alf ИУ</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1302</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
